fcrepo-jms/src/test/java/org/fcrepo/integration/jms/observer/HeadersJMSIT.java

```diff
@@ -131,7 +134,8 @@ public void testIngestion() throws RepositoryException {
         try {
             containerService.findOrCreate(session, testIngested);
             session.save();
-            awaitMessageOrFail(testIngested, NODE_ADDED_EVENT_TYPE, null);
+            final Message ingestMsg = awaitMessageOrFail(testIngested, NODE_ADDED_EVENT_TYPE);
+            assertTrue(getResourceTypes(ingestMsg).contains("fedora:Resource"));
```

Comment:
No, the headers about which we are arguing are in fact the (future) _content_ of that spec. What could possibly be more important to put in an RDF expression of the event than the type of the resource concerned? Unless you are saying that the ref impl should _also_, _additionally_ stick 'em in headers for JMS. I'm not against that, but I don't quite see the utility?